tests/test_sshfsmount.cpp

```diff
@@ -53,14 +53,17 @@ struct SshfsMount : public mp::test::SftpServerTest
         return {std::move(session), default_source, target.value_or(default_target), default_map, default_map};
     }
 
-    auto make_exec_that_fails_for(const std::string& expected_cmd, bool& invoked)
+    auto make_exec_that_fails_for(const std::vector<std::string>& expected_cmds, bool& invoked)
     {
-        auto request_exec = [this, expected_cmd, &invoked](ssh_channel, const char* raw_cmd) {
+        auto request_exec = [this, expected_cmds, &invoked](ssh_channel, const char* raw_cmd) {
             std::string cmd{raw_cmd};
-            if (cmd.find(expected_cmd) != std::string::npos)
+            for (const auto expected_cmd : expected_cmds)
             {
-                invoked = true;
-                exit_status_mock.return_exit_code(SSH_ERROR);
+                if (cmd.find(expected_cmd) != std::string::npos)
+                {
+                    invoked = true;
```

Comment:
This irks me b/c it gets set to `true` if _any_ of the commands are invoked… But probably not something for this PR.

I think `test_command_execution` should be made into something that can also trigger `SSH_ERROR`, at which point all this can be made into a parameterized test.

Reply:
Yeah, I think between this PR and recent merges (and probably upcoming other work), these tests will need refactoring into something better.